test-data-server/src/services/database.interface.ts

```diff
@@ -58,7 +58,7 @@ export interface IDatabase {
 
     getUserForLoginId(loginId: string, userType: string): Promise<string| undefined>;
 
-    getLoginInformation(sector: string): Promise<CustomerModel[] | undefined>;
+    getLoginInformation(sector: string, loginId?: string): Promise<CustomerModel[] | undefined>;
```

Comment:
Shouldn't it be
```suggestion
    getLoginInformation(sector?: string, loginId?: string): Promise<CustomerModel[] | undefined>;
```
?

Reply:
Updated